R.swift/ResourceTypes/LocalizableStrings.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  LocalizableStrings.swift
+//  R.swift
+//
+//  Created by Nolan Warner on 2016/02/23.
+//  Copyright © 2016 Nolan Warner. All rights reserved.
+//
+
+import Foundation
+
+enum Locale {
+  case None
+  case Base
+  case Language(String)
+
+  var isBase: Bool {
+    if case .Base = self {
+      return true
+    }
+
+    return false
+  }
+
+  var isNone: Bool {
+    if case .None = self {
+      return true
+    }
+
+    return false
+  }
+}
+
+extension Locale: CustomStringConvertible {
+  var description: String {
+    switch self {
+    case .None:
+      return ""
```

Comment:
This feel quite tricky, when this is printed in for example a debug sentence you can get very confused by not seeing anything. Can we change this to something like `"None"` or do we need this to be an empty string? If so we should consider not using `CustomStringConvertible`, but another property/method.